src/_includes/layouts/base.njk

```diff
@@ -1,11 +1,21 @@
 <!doctype html>
 <html lang="{{ global.language }}">
   <head>
-    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <title>{{ global.title }}</title>
+    <meta name="description" content="{{ global.description }}">
+    <meta name="author" content="{{ global.author }}">
+    <meta name="twitter:card" value="{{ global.description }}">
+    <meta property="og:title" content="{{ global.title }}">
+    {# Maybe use a different type ? "article" ? #}
+    <meta property="og:type" content="article">
```

Comment:
The "article" value can be used if there is a provided date in the global data file. Otherwise "website" as default? And if it's "article", we can provide `article:published_date` and `article:author`?

Or maybe just stand with the default `website`!